db/migrate/20151024234240_create_students.rb

```diff
@@ -25,14 +25,14 @@ def change
       t.string  :guardian_one_phone_number, null: false
       t.integer :guardian_one_phone_type, null: false
       t.integer :guardian_one_relationship, null: false
-      t.string  :guardian_two_email, null: false
+      t.string  :guardian_two_email, default: '', null: false
       t.string  :guardian_two_employer, default: '', null: false
-      t.string  :guardian_two_first_name, null: false
+      t.string  :guardian_two_first_name, default: '', null: false
       t.string  :guardian_two_job_title, default: '', null: false
-      t.string  :guardian_two_last_name, null: false
-      t.string  :guardian_two_phone_number, null: false
-      t.integer :guardian_two_phone_type, null: false
-      t.integer :guardian_two_relationship, null: false
+      t.string  :guardian_two_last_name, default: '', null: false
+      t.string  :guardian_two_phone_number, default: '', null: false
+      t.integer :guardian_two_phone_type
```

Comment:
Since these are `enums` unfortunately we cannot set a `default: 0` value since `0` actually corresponds to something. I don't have a better fix than to allow `null` right now - will keep thinking!